Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotificationSummary.cs

```diff
@@ -55,5 +55,34 @@ public class SentNotificationSummary
         /// Gets or sets a value indicating whether the sending process is completed or not.
         /// </summary>
         public bool IsCompleted { get; set; }
+
+        /// <summary>
+        /// Gets or sets the sending started date time.
+        /// </summary>
+        public DateTime SendingStartedDateTime { get; set; }
```

Comment:
Change name to SendingStartedDate to align with the entity and the SentDate naming scheme.

Reply:
Addressed. 
I also changed its type to be "DateTime?", to align with the SentDate property.